draft-ietf-tls-tls13.md

```diff
@@ -5846,6 +5846,33 @@ then erase the \[early_]exporter_master_secret, followed by
 each inner values as soon as it is known that it will not be
 needed again.
 
+### Cross-protocol private key re-use
+
+This protocol takes steps (see {{certificate-verify}}) to prevent
+attacks which re-use a private key of the same type across protocols,
+when used for digital signatures.
+
+However, in order to provide an easier migration to the new protocol, and
+for compatibility with older systems which do not fully support new types of
+keys and certificates (e.g., RSASSA-PSS certificates), this protocol does
+not enforce separation of the private key and certificate for different
+protocol versions and usage type. That is, it allows re-using keys marked
+for PKCS#1 1.5 encryption and digital signatures under TLS1.2, for
+RSASSA-PSS signatures under TLS1.3.
```

Comment:
Space between name and number.  "TLS 1.3".

Reply:
thanks